CMakeLists.txt

```diff
@@ -111,6 +111,24 @@ mark_as_advanced(OPENSIM_PYTHON_STANDALONE) # Mainly for packagers; not users.
 option(BUILD_API_ONLY "Build/install only headers, libraries,
 wrapping, tests; not applications (opensim, ik, rra, etc.)." OFF)
 
+option(OPENSIM_DISABLE_LOG_FILE
+"Disable OpenSim from automatically creating 'opensim.log'.
+
+By default, any application linking to osimCommon will create an
+'opensim.log' file in the current working directory. All log messages
+written by OpenSim (incl. during static initialization) are written to
+both this file and the standard output streams.
+
```

Comment:
Nice explanation, but a bit too long to fit on a popup in CMake.  Can we condense to match other options, second paragraph could suffice.